src/main/extras/app/Cryostat.xml

```diff
@@ -0,0 +1,1200 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+     Recommended way to edit .jfc files is to use Java Mission Control,
+     see Window -> Flight Recorder Template Manager.
+-->
+<configuration version="2.0" label="Cryostat" description="Low overhead configuration safe for continuous use in production environments, typically less than 1 % overhead." provider="Oracle">
```

Comment:
Could you replace the `description` here with some short blurb about this being used for Cryostat self-profiling?

Also, please update the `provider` to Cryostat.